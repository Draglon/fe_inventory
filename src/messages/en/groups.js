const groups = {
  "Groups": {
    "layout": {
      "title": "Groups",
      "description": "Groups - description",
    },
    "title": "Groups",
    "addProduct": "Add product",
  },
};

export default groups;