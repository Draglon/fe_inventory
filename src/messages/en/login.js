const login = {
  "Login": {
    "layout": {
      "title": "Login",
      "description": "Login - description",
    },
    "title": "Login",
  },
};

export default login;