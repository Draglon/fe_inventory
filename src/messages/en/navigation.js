const navigation = {
  "Navigation": {
    "parishes": "Parishes",
    "groups": "Groups",
    "products": "Products",
    "users": "Users",
    "settings": "Settings"
  },
};

export default navigation;