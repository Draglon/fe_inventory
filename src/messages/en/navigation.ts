const navigation = {
  "Navigation": {
    "orders": "Orders",
    "groups": "Groups",
    "products": "Products",
    "users": "Users",
    "settings": "Settings"
  },
};

export default navigation;