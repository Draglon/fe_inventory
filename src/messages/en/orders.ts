const orders = {
  "Orders": {
    "layout": {
      "title": "Orders page",
      "description": "Orders page - description",
    },
    "title": "Orders",
  },
};

export default orders;