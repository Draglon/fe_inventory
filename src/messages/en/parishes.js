const parishes = {
  "Parishes": {
    "layout": {
      "title": "Parishes page",
      "description": "Parishes page - description",
    },
    "title": "Parishes",
  },
};

export default parishes;