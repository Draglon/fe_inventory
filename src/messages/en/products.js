const products = {
  "Products": {
    "layout": {
      "title": "Products page",
      "description": "Products page - description",
    },
  },
};

export default products;