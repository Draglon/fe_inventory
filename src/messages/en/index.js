import login from "./login";
import signup from "./signup";
import groups from "./groups";
import orders from "./orders";
import products from "./products";
import settings from "./settings";
import users from "./users";
import navigation from "./navigation";
import localeSwitcher from "./localeSwitcher";
import shared from "./shared";

// eslint-disable-next-line import/no-anonymous-default-export
export default {
  ...login,
  ...signup,
  ...groups,
  ...orders,
  ...products,
  ...settings,
  ...users,
  ...navigation,
  ...localeSwitcher,
  ...shared,
};