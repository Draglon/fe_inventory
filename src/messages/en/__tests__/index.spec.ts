import en from "..";

it("en locale matches snapshot", () => {
  expect(en).toMatchSnapshot();
});
