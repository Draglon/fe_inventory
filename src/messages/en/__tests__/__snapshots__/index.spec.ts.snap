// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`en locale matches snapshot 1`] = `
{
  "Groups": {
    "addProduct": "Add product",
    "layout": {
      "description": "Groups - description",
      "title": "Groups",
    },
    "title": "Groups",
  },
  "LocaleSwitcher": {
    "label": "Change language",
    "locale": "{locale, select, ru {Russian} ua {Ukrainian} en {English} other {Unknown}}",
  },
  "Login": {
    "layout": {
      "description": "Login - description",
      "title": "Login",
    },
    "title": "Login",
  },
  "Navigation": {
    "groups": "Groups",
    "orders": "Orders",
    "products": "Products",
    "settings": "Settings",
    "users": "Users",
  },
  "Orders": {
    "layout": {
      "description": "Orders page - description",
      "title": "Orders page",
    },
    "title": "Orders",
  },
  "Products": {
    "filter": {
      "specification": {
        "label": "Specification:",
        "option": {
          "all": "All",
          "monitor": "Monitor",
          "phone": "Phone",
        },
      },
      "type": {
        "label": "Type:",
        "option": {
          "all": "All",
          "monitor": "Monitor",
          "phone": "Phone",
        },
      },
    },
    "guarantee": {
      "from": "from",
      "to": "to",
    },
    "layout": {
      "description": "Products page - description",
      "title": "Products page",
    },
    "title": "Products",
  },
  "Registration": {
    "description": "Please, fill all fields",
    "form": {
      "email": {
        "errors": {
          "required": "Please input your email!",
        },
        "label": "Email",
        "placeholder": "Input email",
      },
      "password": {
        "errors": {
          "maxLength": "Must be no more than 20 characters",
          "minLength": "Must be at least 6 characters long",
          "required": "Please input your password!",
        },
        "label": "Password",
        "placeholder": "Input password",
      },
      "userName": {
        "errors": {
          "required": "Please input your username!",
        },
        "label": "Username",
        "placeholder": "Input username",
      },
    },
    "layout": {
      "description": "Registration - description",
      "title": "Registration",
    },
    "title": "Sign up",
  },
  "Settings": {
    "layout": {
      "description": "Settings - description",
      "title": "Settings",
    },
    "title": "Settings",
  },
  "Users": {
    "layout": {
      "description": "Users - description",
      "title": "Users",
    },
    "title": "Users",
  },
  "shared": {
    "add": "Add",
    "cancel": "Cancel",
    "confirm": "Confirm",
    "create": "Create",
    "delete": "Delete",
    "email": "Email",
    "logIn": "Log in",
    "logout": "Log out",
    "modal": {
      "removeOrder": {
        "title": "Are you sure you want to remove this Order?",
      },
      "removeParish": {
        "title": "Are you sure you want to remove this parish?",
      },
      "removeProduct": {
        "title": "Are you sure you want to remove this product?",
      },
    },
    "next": "Next",
    "password": "Password",
    "previous": "Previous",
    "profile": "Profile",
    "save": "Save",
    "search": "Search",
    "settings": "Settings",
    "signUp": "Sign up",
    "status": {
      "free": "In stock",
      "new": "New",
      "repair": "Repair",
      "reserved": "Reserved",
      "sold": "Sold",
      "used": "Used",
    },
  },
}
`;
